prometheus/examples_test.go

```diff
@@ -319,6 +319,8 @@ func ExampleSummary() {
 	// internally).
 	metric := &dto.Metric{}
 	temps.Write(metric)
+	// We remove CreatedTimestamp just to make sure the assert below works.
+	metric.Summary.CreatedTimestamp = nil
```

Comment:
Alternatively we could mock `now: func() time.Time { return time.Time{} }` and use `0001-01-01 00:00:00 +0000 UTC` string (or similar in proto)

Reply:
but `now` is an unexported attribute of `SummaryOpts` since the only goal of the now function is to enable good tests (we don't want people setting weird functions in their instrumentation).

examples_test lives in a different package, therefore we can't set a mocked now function.